src/components/field/FieldBody.vue

```diff
@@ -0,0 +1,14 @@
+<script>
+    export default {
+        name: 'fieldBody',
+        render(h) {
+            return h('div', { attrs: { 'class': 'field-body' }}, this.$slots.default.map(v => {
+                // skip returns and comments
+                if (!v.tag) {
+                    return v
+                }
+                return h('div', { attrs: { 'class': 'field' }}, [v])
```

Comment:
Lets use template here instead of render function to keep a pattern.

Reply:
I'll be happy to do that, if only I knew how...
The whole (and unique) purpose of this FieldBody component is to:
* enclose everything inside a `<div class="field-body">` tag
* wrap each `<slot/>` entries in a `<div class="field">`

Not doing the second point means we can't have more than one control in a horizontal field, while the [Bulma doc](https://bulma.io/documentation/form/general/#horizontal-form) show it is possible.

In fact, I've started without any FieldBody, using this template:

    <div class="field-body" v-if="horizontal">
        <div class="field"><slot></slot></div>
    </div>


But it doesn't works with more than one control in slot.

Reply:
What's the difficult on doing that with template syntax? Not sure if I got the point.

Reply:
All I got with a template solution was:

    <div class="field-body">
        <div class="field">
            <input/>
            <input/>
            ...
        </div>
    </div

But what is really reeded is:

    <div class="field-body">
        <div class="field">
            <input/>
        </div>
        <div class="field">
            <input/>
        </div>
    </div>

And I really don't know how to do that with template only. 

Found this render solution on [stackoverflow](https://stackoverflow.com/questions/43834785/how-add-parent-for-each-slots-in-vuejs)

Reply:
You are right, this is not easily done without render functions, my mistake.